src/Swarm/Language/Parse.hs

```diff
@@ -98,6 +98,7 @@ reservedWords =
        , "false"
        , "forall"
        , "require"
+       , "atomic"
```

Comment:
Need to update the editor support/ syntax highlighting

Reply:
Seems like you already did that? :slightly_smiling_face:

Reply:
Indeed!